apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -0,0 +1,208 @@
+				-ms-grid-column-span: 1;
+				grid-column-end: span 1;
+				height: 100vh;
+				justify-content: space-between;
+			}
+			.main,
+			.marketing {
+				flex: 0 0 100%;
+			}
+			.name {
+				color: #fff;
+				font-size: 19px;
+				line-height: 1.3;
+				margin-bottom: 8px;
+				padding: 0;
+				text-align: left;
+			}
+			.description {
+				color: #fff;
+				font-size: 40px;
+				line-height: 1.15;
+				padding: 0;
+				text-align: left;
+			}
+			.description,
+			.copy {
+				font-family: Georgia, "Times New Roman", Times, serif;
+			}
+			.marketing {
+				padding-bottom: 8px;
+			}
+			.marketing-copy {
+				display: flex;
+				align-items: center;
+			}
+			.logo {
+				height: 32px;
+				margin-right: 16px;
+				width: 32px;
+			}
+			.copy {
+				line-height: 1.4;
+				margin: 0;
+			}
+			.marketing-buttons .button {
+				background: #fff;
+				border-radius: 2px;
+				border: 1px solid #fff;
+				box-sizing: border-box;
+				color: #117ac9;
+				display: block;
+				font-size: 16px;
+				font-weight: 700;
+				line-height: 21px;
+				padding: 13px;
+				text-align: center;
+				text-overflow: ellipsis;
+				text-decoration: none;
+				transition: opacity .15s ease-out;
+				white-space: nowrap;
+				width: 100%;
+			}
+			.marketing-buttons .button-secondary,
+			.marketing-buttons .button-secondary:hover,
+			.marketing-buttons .button-secondary:focus {
+				background: transparent;
+				color: #fff;
+			}
+			.marketing-buttons .button:hover,
+			.marketing-buttons .button:focus {
+				opacity: .85;
+			}
+			@media screen and ( min-width: 660px ) {
+				.description,
+				.copy {
+					font-family: Recoleta, Georgia, "Times New Roman", Times, serif;
+				}
+				.name {
+					font-size: 23px;
+				}
+				.description {
+					font-size: 69px;
+				}
+				.marketing {
+					align-items: center;
+					display: flex;
+					justify-content: space-between;
+					padding-bottom: 24px;
+				}
+				.marketing-copy {
+					margin-right: 16px;
+				}
+				.marketing-buttons {
+					display: flex;
+				}
+				.marketing-buttons p {
+					margin: 0;
+				}
+				.marketing-buttons p:nth-child(2) {
+					margin-left: 8px;
+				}
+				.marketing-buttons .button {
+					font-size: 13px;
+					padding: 7px 13px;
+					min-width: 145px;
+				}
+			}
+			@media screen and ( min-width: 661px ) {
+				p, ul, ol {
+					font-size: 16px;
+					letter-spacing: 0;
+				}
+			}
+			@media screen and ( min-width: 960px ) {
+				.name {
+					font-size: 28px;
+					margin-bottom: 16px;
+				}
+				.description {
+					font-size: 99px;
+				}
+				.copy {
+					font-size: 19px;
+				}
+			}
+			@media screen and ( min-width: 1040px ) {
+				.body {
+					-ms-grid-columns: (1fr)[12];
+					grid-template-columns: repeat(12, 1fr);
+				}
+				.inner {
+					-ms-grid-column: 2;
+					grid-column-start: 2;
+					-ms-grid-column-span: 10;
+					grid-column-end: span 10;
+				}
+				.marketing {
+					padding-bottom: 32px;
+				}
+			}
+		</style>
```

Comment:
Since this is quite a bit of inline CSS, it might make sense to have it as a separate file and put through build-pipeline. Then either enqueue it as a file or inject as inline CSS.
 
Benefits:
- You get linting
- You can use sass, which would be important for using variables to keep things more consistent. 
- You get Autoprefixer so no human errors with vendor specific CSS
- You get minification (could be we do that for inline CSS but I haven't checked)

Reply:
Great idea! For the moment, I just copied over the existing coming soon page verbatim from wpcom, but it'd be good to fix all this up. There's some styling overrides we'll likely want to add too, to make sure that theme styling doesn't mess with the login buttons on the mobile view, too.